test/indexer/test_tdr.py

```diff
@@ -147,18 +161,42 @@ def test_fetch_bundle(self):
         links_content_column[0] = json.dumps(links_content)
         # Invoke code under test
         with self.assertRaises(RequirementError):
-            self._test_fetch_bundle(self.source, load_tables=False)
+            self._test_fetch_bundle(bundle, load_tables=False)
+
+    def test_subgraph_stitching(self):
+        bundle = self._canned_bundle(self.source,
+                                     '4426adc5-b3c5-5aab-ab86-51d8ce44dfbe')
+        assert any(e['is_stitched'] for e in bundle.manifest)
+        source_name = bundle.fqid.source.name
+        plugin = self.plugin_for_source_name(source_name)
+
+        self._test_fetch_bundle(bundle, load_tables=True)
+
+        # Remove upstream bundles and verify failure to follow broken links
+        downstream = list(plugin._run_sql(f"""
+            SELECT *
```

Comment:
I would blatantly hard-code a deletion from the list at `plugin.tinyquery.tables['links'].columns[*].values`, after maybe asserting that `plugin.tinyquery.tables['links'].columns['links_id'].values[0]`  is the uuid of the upstream bundle.

Reply:
There are two upstream bundles in this test, so instead of multiple removals I just set the columns to singleton lists containing only the downstream bundle.

Reply:
Is there a reason to prefer the assert-based approach over dynamically choosing which rows to remove?

Reply:
Yes, brevity. If you can show me that the dynamic approach is actually less code, I'd be ok with it.